src/Sign.Cli/CodeCommand.cs

```diff
@@ -30,11 +30,11 @@ internal CodeCommand()
         {
             DescriptionOption.IsRequired = true;
             DescriptionUrlOption.IsRequired = true;
-            TimestampUrlOption.IsRequired = true;
 
             MaxConcurrencyOption.SetDefaultValue(4);
             FileDigestOption.SetDefaultValue(HashAlgorithmName.SHA256);
             TimestampDigestOption.SetDefaultValue(HashAlgorithmName.SHA256);
+            TimestampUrlOption.SetDefaultValue(new Uri("http://timestamp.acs.microsoft.com"));
```

Comment:
Is this expected to be stable service after many years?  If I understood correctly actual certificates usually expire after few years, then this one is kick in,

Reply:
It is expected to be stable.  The URL is separate from which signing certificate(s) are used.  The signing certificates rotate over time, while the URL should be stable.